stores/user.ts

```diff
@@ -53,11 +60,39 @@ export const useUserStore = defineStore('user', () => {
     }
   }
 
+  async function fetchLikerInfo ({ nocache = false } = {}) {
```

Comment:
```suggestion
  async function fetchLikerProfile ({ nocache = false } = {}) {
```
Or any other name that we won't confuse this with `fetchLikerInfoById`